### PR TITLE
Added some data to sanity for the about section.

### PR DESCRIPTION
**Pull request by ronaldpaek**

### CODE REVIEW

File: src/pages/About.js [hunk 1]
```diff
@@ -9,15 +9,14 @@ import { style } from '../styles';
...
+	console.log(skills);
 
 	useEffect(() => {
-		const query = `*[_type == 'abouts']`;
+		const query = `*[_type == 'skills']`;
 
 		client.fetch(query).then(data => {
-			console.log('hello');
-			setAbouts(data);
+			setSkills(data);
```
Comment (TidbitsJS) [inline src/pages/About.js:19]: Move the sanity fetching functions to a separate file. Import and call them accordingly.

**Always, always catch the errors while doing any fetch logic**. Using the try-catch block with async-await will be better in my eyes. Not mandatory, though.

In the future, we will catch and display these errors on UI. For now, simply console log it.

**Review by TidbitsJS — CHANGES_REQUESTED** [review]

**Comment by ronaldpaek** [comment]
Updated Theme object to work with dark mode, implemented color schemes, reworked using after effect, reworked logic to toggle zIndex so H2 shows under menuBar, setup first 2 pages to toggle with color scheme, refactor all code to be consistent with MUI style like you told me. I just need to break down each component then it's manageable, when it's too long of code I get lost so I adjust css on the component with sx prop and utilize the theme object / function to be able to use color scheme add new colors tested it out did style overrides removed css file, also I figured out how to use flex: 1 and tested out mui grid system, I just don't like it, although I made, but I actually really prefer just using grid now, I will do the rest of the pages.

**Comment by ronaldpaek** [comment]
Refactored my context to custom hooks useColorModeContext and useOpenModeContext, since I made context and broke down my components I realize I don't need to pass any functions down as props anymore, refactor to just useContext whenever the component in the component itself. Refactor my ColorModeContext Provider so I can more easily modify the theme object if needed, made separate file and created a theme and imported it to the provider.
Move components into Folders to group subcomponents together for the specific page. Refactor to break down code into more components so it's easier and less stressful to read code, decided to use display flex or grid only now, so no more mui grid, realized it doesn't work the way I like and also display: grid is a lot easier to implement, but played around with flex: 1 etc refactor to make my code look consistent code style and using good semantics along with using clean mui techniques.

File: src/components/About/Clients.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+import { brands } from '../../constants';
+
+const Clients = () => {
+	return (
+		<Box>
+			<Box
+				sx={{
+					bgcolor: theme =>
+						theme.palette.mode === 'light' ? '#F8F8F8' : '#0D0D0D',
```
Comment (TidbitsJS) [inline src/components/About/Clients.js:13]: Can we move these colors to the custom color palette you have created? 

```javascript
bgcolor: theme  => theme.custom.bg.primary
```

Something like that you have done it for `color: theme => theme.custom.neutral.primary` field.

File: src/components/Header/ThemeModeToggleButton.js [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+			sx={theme => ({
+				display:
+					displayMode === 'mobile'
+						? { xs: 'inline-flex', lg: 'none' }
+						: { xs: 'none', lg: 'inline-flex' },
+				width: 44,
+				height: 44,
+				ml: 2,
+				bgcolor: theme.custom.toggle.bgcolor,
+				color: theme.palette.mode === 'light' ? 'black' : 'white',
```
Comment (TidbitsJS) [inline src/components/Header/ThemeModeToggleButton.js:20]: Use your custom theme

File: src/components/Resume/ColumnItem.js [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+					borderWidth: 2
+				})
+			})}
+		>
+			<Typography
+				component='span'
+				sx={{
+					fontSize: '.875rem',
+					color: theme =>
+						theme.palette.mode === 'light' ? '#44566C' : '#B7B7B7'
```
Comment (TidbitsJS) [inline src/components/Resume/ColumnItem.js:26]: Use your custom theme

File: src/components/Resume/ColumnItem.js [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+				sx={{
+					color: theme => theme.custom.neutral,
+					fontSize: '1.25rem'
+				}}
+			>
+				{title}
+			</Typography>
+			<Typography
+				sx={{
+					color: theme => (theme.palette.mode === 'light' ? 'black' : '#B7B7B7')
```
Comment (TidbitsJS) [inline src/components/Resume/ColumnItem.js:42]: Use the custom theme

File: src/components/Resume/Knowledges.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+			>
+				{buttons.map((title, i) => (
+					<Box
+						key={i}
+						component='button'
+						sx={{
+							borderRadius: '.5rem',
+							color: 'secondary.main',
+							bgcolor: theme =>
+								theme.palette.mode === 'light' ? '#EDF2F2' : '#1C1C1C',
```
Comment (TidbitsJS) [inline src/components/Resume/Knowledges.js:34]: Same. Move those colors to the custom theme and use it here as you used on the rest places

File: src/components/Resume/WorkingSkills.js [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+			</Typography>
+			{skills.map(({ title, value, color }, i) => (
+				<Box key={i} mb={5}>
+					<Stack direction='row' justifyContent='space-between' mb={1}>
+						<Box
+							component='span'
+              sx={{
+                fontWeight: 600,
+								color: theme =>
+									theme.palette.mode === 'light' ? '#526377' : '#A6A6A6'
```
Comment (TidbitsJS) [inline src/components/Resume/WorkingSkills.js:28]: Use custom theme

File: src/components/Resume/WorkingSkills.js [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+							}}
+						>
+							{title}
+						</Box>
+						<Box
+							component='span'
+              sx={{
+                fontWeight: 600,
+								color: theme =>
+									theme.palette.mode === 'light' ? '#526377' : '#A6A6A6'
```
Comment (TidbitsJS) [inline src/components/Resume/WorkingSkills.js:38]: Use custom theme

**Review by TidbitsJS — CHANGES_REQUESTED** [review]

**Review by TidbitsJS — APPROVED** [review]

**Pull request merged**